packages/webpack-config-single-spa-react/lib/webpack-config-single-spa-react.js

```diff
@@ -3,10 +3,14 @@ const webpackConfigSingleSpa = require("webpack-config-single-spa");
 module.exports = webpackConfigSingleSpaReact;
 
 function webpackConfigSingleSpaReact(opts) {
+  const webpackConfigEnv = opts.webpackConfigEnv || {};
+
   opts.react = true;
   const config = webpackConfigSingleSpa(opts);
 
-  config.externals.push("react", "react-dom");
+  if (!webpackConfigEnv.standalone) {
```

Comment:
When in standalone mode, there's no need to use react/react-dom from the import map